frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -67,12 +67,15 @@ const Metric = (props: MetricProps) => {
     const theme = useTheme();
 
     const data = useMemo(() => {
```

Comment:
would be nice to type the data as Data and layout as Partial<Layout>
And add title to layout ;-)